src/modules/numbers/numbers.controller.ts

```diff
@@ -0,0 +1,41 @@
+import { Body, Controller, Get, Query, Post, UseInterceptors, CacheInterceptor } from '@nestjs/common';
+import {
+  ApiBearerAuth,
+  ApiBody,
+  ApiOperation,
+  ApiResponse,
+  ApiTags,
+} from '@nestjs/swagger';
+import { NumbersService } from './numbers.service';
+import { CreateNumberDto } from './dto/create-number.dto';
+import { UkefNumber } from './entities/number.entity';
+
+@ApiBearerAuth()
+@ApiTags('numbers')
+@Controller('numbers')
+export class NumbersController {
+  constructor(private readonly numberService: NumbersService) {}
+
+  @Post()
+  @ApiOperation({ summary: 'Create Number' })
+  @ApiBody({ type: [CreateNumberDto]})
+  @ApiResponse({ status: 201, description: 'Created.' })
+  async create(@Body() ukefIDRecord: CreateNumberDto[]): Promise<UkefNumber[]> {
+    const createResult = await this.numberService.create(ukefIDRecord);
```

Comment:
do we need to validate the ukefIDRecord here? we can use express validator or joi.js if needed

Reply:
because there are no transformations here, we can return the function straight away
so something like:
`return this.numberService.create(ukefIdRecord);`
(no need for `await` if we return it straight away)

Reply:
Removed await and sync.

NestJs is doing validation against CreateNumberDto class. It required ParseArrayPipe to get it working with array. 
We could use other validation libraries in future if required.